peering/peering.go

```diff
@@ -187,6 +187,19 @@ func (ps *PeeringService) Start() error {
 	return nil
 }
 
+//	GetState get the State of the Peering Service
+func (ps *PeeringService) GetState() string {
```

Comment:
That racy it need lock unlock ps.mu.

Reply:
> Thx for you patch. I'm not sure it is usefull. But if you tell me you used it in some other code that fine for me.
> 
> I would like to see a few things changed first pls.

I will be using it on estuary https://github.com/application-research/estuary/pull/260. It's for a new endpoint to allow the consumer to check the state of the `peering`.